Examples/ObjectiveC/ShowHideLayerExample.m

```diff
@@ -79,14 +79,25 @@ - (void)hideContours {
 
 - (void)addToggleButton {
     UIButton *button = [UIButton buttonWithType:UIButtonTypeSystem];
-    button.autoresizingMask = UIViewAutoresizingFlexibleTopMargin | UIViewAutoresizingFlexibleLeftMargin | UIViewAutoresizingFlexibleRightMargin;
     [button setTitle:@"Toggle Contours" forState:UIControlStateNormal];
     [button setSelected:YES];
     [button sizeToFit];
     button.center = CGPointMake(self.view.center.x, 0);
     button.frame = CGRectMake(button.frame.origin.x, self.view.frame.size.height - button.frame.size.height - 5, button.frame.size.width, button.frame.size.height);
     [button addTarget:self action:@selector(toggleLayer:) forControlEvents:UIControlEventTouchUpInside];
     [self.view addSubview:button];
+    
+    if (@available(iOS 11, *)) {
+        UILayoutGuide *safeArea = self.view.safeAreaLayoutGuide;
+        NSArray *constraints = @[
+          [button.bottomAnchor constraintEqualToAnchor: safeArea.bottomAnchor constant:-5],
```

Comment:
o_O I'm not really sure what is causing this but it seems like the alignment doesn't match with the Obj-C and Swift examples for the bottom anchor even though the values are the same...?

<img width="895" alt="screenshot 2019-03-01 15 47 41" src="https://user-images.githubusercontent.com/10850812/53672981-df612980-3c39-11e9-912a-d1e7c49ff2db.png">

_Left is Obj-C, right is Swift_